cellar-tags/tagging.md

```diff
@@ -86,11 +86,14 @@ Although tags are metadata mostly used for reading, there are cases where the st
 be used for sorting, categorization, etc. For this reason, when possible, strict formatting
 of the value should be used so everyone can agree on how to use the value.
 
-Multiple items **MUST NOT** be stored as a list in a single `TagString`. If there is more
-than one tag value with the same name to be stored, then more than one `SimpleTag` **MUST** be used.
+Multiple items **SHOULD NOT** be stored as a list in a single `TagString`. If there is more
+than one tag value with the same name to be stored,
+it is **RECOMMENDED** to use separate `SimpleTag`s with that name for each value.
```

Comment:
```suggestion
Multiple items **SHOULD NOT** be stored as a list in a single `TagString`. If there is more
than one tag value with the same name to be stored,
it is **RECOMMENDED** to use separate `SimpleTag`s with that name for each value.
```
Is this (the case we so often have) recognizing that things from the past don't conform to our current rules, and one must be prepared to handle currently-invalid cases from the past, but ought not be carried forward in the future? 

If so, I'd be more comfortable if we said that. **RECOMMENDED** means it will be more difficult to justify adopting our new practice. 

Do The Right Thing, of course.